lib/airbrake-ruby/async_sender.rb

```diff
@@ -24,7 +24,12 @@ def initialize(config)
     #   library
     # @return [nil]
     def send(notice)
-      @unsent << notice
+      if @unsent.size >= @unsent.max
+        msg = "#{LOG_LABEL} dropping notice since sending queue is full"
+        @config.logger.error msg
```

Comment:
It was more of a question. I am not sure if we need to log, but maybe you would find it useful?
If we do want, I am not sure about the error severity. It's more like a `warn` to me, but I am open to other suggestions.